WordPress/gutenberg
# feat: Announce offline status

## What?

Announce offline status when network connectivity is lost.

## Why?

Fix https://github.com/wordpress-mobile/gutenberg-mobile/issues/6475. Improve
the UX for users relying upon screen readers.

## How?

Implement a React Hook for replicating "live regions" for both iOS and Android.
Announce a message whenever the network connection status changes.

## Testing Instructions




1. Enable your device's screen reader.
1. Launch the editor.
1. Verify the network connectivity status is _not_ audibly announced.
1. Disable your device's network connection.
1. Verify the network connectivity status is audibly announced accurately.
1. Enable your device's network connection.
1. Verify the network connectivity status is audibly announced accurately.

### Testing Instructions for Keyboard

1. Enable your device's screen reader.
1. Launch the editor.
1. Disable your device's network connection.
1. Navigate focus to the "Working offline" alert. 
1. Verify a helpful message is read aloud.

## Screenshots or screencast 
N/A, no visible changes proposed.
